data/bots/lib/libbot/geo-feature-specifications.ts

```diff
@@ -24,7 +24,7 @@ export class GeoFeatureSpecifications {
     };
 
     return await this.baseHttpClient.post(
-      `/scenarios/${scenarioId}/features/specification/v2`,
+      `/scenarios/${scenarioId}/features/specification`,
```

Comment:
ah sorry, forgot about this place when was changing!

Reply:
well, *I* should have known better, also because it was me who pushed to drop the `v2` 😅